### PR TITLE
Rename GeanySendMail to plain SendMail

### PR DESCRIPTION
**Pull request by frlan**
This pull request shall clean up name of plugins starting with Geany-prefix a little.

### CODE REVIEW

**Comment by codebrainz** [comment]
Looks OK on a quick skim. +1 for removing prefix.

File: sendmail/README [hunk 1]
```diff
@@ -1,16 +1,18 @@
-GeanySendMail
+SendMail
 =============
```
Comment (b4n) [inline sendmail/README]: ideally underlining length should match new title

File: sendmail/README [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 About
 ------
 
-GeanySendMail is a little plugin to send a document as attachment
+SendMail is a little plugin to send a document as attachment
 using the preferred mail client from inside Geany. It is similar to
 the envelope symbol of most office tools and requires a mail client
 that is supporting remote calls.
 
+This is not a direct binding to sendmail, even it might could be used for.
```
Comment (b4n) [inline sendmail/README]: *even if it can be used as such**

File: sendmail/src/sendmail.c [hunk 3]
```diff
@@ -354,12 +359,69 @@ GtkWidget *plugin_configure(GtkDialog *dialog)
...
+	gchar *config_dir_old = NULL;
 	gchar *kb_label = _("Send file by mail");
 	GtkWidget *menu_mail = NULL;
 	GeanyKeyGroup *key_group;
 
 	config_file = g_strconcat(geany->app->configdir, G_DIR_SEPARATOR_S, "plugins", G_DIR_SEPARATOR_S,
+		"sendmail", G_DIR_SEPARATOR_S, "mail.conf", NULL);
+
+	#ifndef G_OS_WIN32
+	/* We try only to move if we are on not Windows platform */
```
Comment (b4n) [inline sendmail/src/sendmail.c]: Why not on Windows?

Comment (frlan) [inline sendmail/src/sendmail.c]: I just had no chance to test and didn't want to make an guessing.

**Comment by b4n** [comment]
Apart that, LGBI

**Pull request merged**